packages/play-button/src/index.ts

```diff
@@ -111,15 +111,21 @@ if (!specUrlElement && !specElement && !specScriptTag) {
         if (specifiedOperation) {
           open({
             path: specifiedOperation.path,
-            method: specifiedOperation.httpVerb,
+            method: specifiedOperation.httpVerb.toLowerCase() as Exclude<
```

Comment:
```suggestion
            method: specifiedOperation.httpVerb.toLowerCase() as Exclude<
```
instead of as we like to use type guards now for safety. We have a few around like
https://github.com/scalar/scalar/blob/main/packages/oas-utils/src/helpers/httpMethods.ts#L85